tensorflow/compiler/mlir/tosa/transforms/passes.td

```diff
@@ -60,6 +60,12 @@ def TosaStripQuantTypesPass : Pass<"tosa-strip-quant-types", "mlir::func::FuncOp
   let dependentDialects = ["TosaDialect"];
 }
 
+def TosaStripComplexTypesPass : Pass<"tosa-strip-complex-types", "mlir::func::FuncOp"> {
+  let summary = "Convert all complex types to their storage type.";
+  let constructor = "createStripComplexTypesPass()";
+  let dependentDialects = ["TosaDialect"];
```

Comment:
Could you perhaps add description here too? (I know we don't yet generate docs for these ... but hopefully one day :)).

"Strip" feels a bit off in the name. Lower/flatten ?

Reply:
Agreed on adding a description and happy to change the name, how about "replace"? I feel "lower" might be a bit less specific and "flatten" to me indicates a change in tensor shape but not a change in data type. Fully open to other opinions/suggestions though!

Reply:
I'm good with lower thanks.

Reply:
Apologies, just to confirm are you happy with "replace", or would you prefer "lower"?

Reply:
SGTM thanks